wiki/Beatmap/Difficulty/id.md

```diff
@@ -57,17 +66,17 @@ Berikut adalah nama tingkat kesulitan yang biasa dimiliki dalam beatmap. Definis
 - ![](/wiki/shared/diff/expert-m.png) Expert
 - ![](/wiki/shared/diff/expertplus-m.png) Expert+
 
-## Star rating (tingkat kesulitan dalam skala bintang)
+## Star rating
 
 ![Jangkauan star rating](img/SR-range.png)
 
-Di situs web, beatmap secara otomatis diberikan ikon kesulitan berdasarkan rentang *star rating* ini:
+Situs web osu! secara otomatis mengelompokkan beatmap-beatmap yang diunggah ke dalam enam rentang tingkat kesulitan berdasarkan star rating dari beatmap yang bersangkutan sebagai berikut:
 
 - ![](/wiki/shared/diff/easy-s.png) Easy: 0.0★–1.99★
 - ![](/wiki/shared/diff/normal-s.png) Normal: 2.0★–2.69★
 - ![](/wiki/shared/diff/hard-s.png) Hard: 2.7★–3.99★
 - ![](/wiki/shared/diff/insane-s.png) Insane: 4.0★–5.29★
 - ![](/wiki/shared/diff/expert-s.png) Expert: 5.3★–6.49★
-- ![](/wiki/shared/diff/expertplus-s.png) Expert+: 6.5★ dan di atasnya
+- ![](/wiki/shared/diff/expertplus-s.png) Expert+: 6.5★ atau lebih
 
-Tingkat kesulitan yang dimiliki dan diberikan pada sebagian besar beatmap cocok dengan ikon yang diberikan, tetapi beberapa menyimpang bahkan secara signifikan. Inilah sebabnya mengapa [kriteria beatmap Ranked](/wiki/Ranking_Criteria) mendefinisikan aturan yang lebih akurat untuk menentukan kesulitan dan mengavaluasi semua set yang ada dalam beatmap.
+Star rating suatu beatmap pada umumnya cukup menggambarkan seberapa sukar beatmap yang bersangkutan, namum dalam berbagai kasus terdapat beatmap-beatmap yang pada kenyataannya jauh lebih mudah ataupun lebih sukar dibanding star rating yang tertera. Inilah sebabnya mengapa [Ranking Criteria](/wiki/Ranking_Criteria) mendefinisikan aturan yang lebih spesifik untuk menentukan tingkat kesulitan yang sesuai untuk suatu beatmap.
```

Comment:
```suggestion
Star rating suatu beatmap pada umumnya cukup menggambarkan seberapa sukar beatmap yang bersangkutan. Namun, dalam berbagai kasus, terdapat beatmap-beatmap yang pada kenyataannya jauh lebih mudah ataupun lebih sukar dibanding star rating yang tertera. Inilah mengapa [Ranking Criteria](/wiki/Ranking_Criteria) mendefinisikan aturan yang lebih spesifik untuk menentukan tingkat kesulitan yang sesuai untuk suatu beatmap.
```
`namun` is an inter-sentence conjunction
`sebabnya mengapa` is ineffective. you may choose which one to delete, but I prefer the former